docs/Bots.md

```diff
@@ -305,62 +305,68 @@ The modify expert bot allows you to change arbitrary field values of events just
 The configuration is called `modify.conf` and looks like this:
 
 ```json
-{
-"Spamhaus Cert": {
-    "__default": [{
-            "feed.name": "^Spamhaus Cert$"
-        }, {
-            "classification.identifier": "{msg[malware.name]}"
-        }],
-    "conficker": [{
-            "malware.name": "^conficker(ab)?$"
-        }, {
+[
+    {
+        "rule": "Standard Protocols http",
```

Comment:
let's call this "rule-name" or "rule name".
It's important for the reader to know that this might be ignored.